pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/config/TableDataManagerConfig.java

```diff
@@ -41,6 +41,8 @@
   private static final String READ_MODE = "readMode";
   private static final String TABLE_DATA_MANAGER_DATA_DIRECTORY = "directory";
   private static final String TABLE_DATA_MANAGER_NAME = "name";
+  private static final String REALTIME_AVG_MULTI_VALUE_COUNT = "avgMvCount";
```

Comment:
Do we want count here or length? Would "realtime average multi value entries per row" be a more appropriate description?